extensions/libarchive/FocusArchiveEntry.cpp

```diff
@@ -295,7 +260,9 @@ int64_t FocusArchiveEntry::ReadCallback::process(std::shared_ptr<io::BaseStream>
   return nlen;
 }
 
-FocusArchiveEntry::ReadCallback::ReadCallback(core::Processor *processor,fileutils::FileManager *file_man, ArchiveMetadata *archiveMetadata)
```

Comment:
Our line length is 200. We haven't merged your changes to lint the extensions but you may want to limit these types of changes to reduce later changes.